conf/docker-dcm/c8client.dockerfile

```diff
@@ -0,0 +1,7 @@
+# build from repo root
```

Comment:
Should we delete c6client.dockerfile since we're adding c8client.dockerfile?

Reply:
Probably - seems like it's less confusing to have two copies.

Reply:
In the end we decided to stick with the older, already tested version of CentOS for DCM stuff.